engine/class_modules/paladin/sc_paladin.cpp

```diff
@@ -3052,6 +3115,8 @@ double paladin_t::resource_loss( resource_e resource_type, double amount, gain_t
   double result     = player_t::resource_loss( resource_type, amount, source, action );
   if ( resource_type == RESOURCE_HOLY_POWER && result > 0 && legendary.of_dusk_and_dawn->ok() &&
        resources.current[ RESOURCE_HOLY_POWER ] == legendary.of_dusk_and_dawn->effectN( 2 ).base_value() )
+  if ( resource_type == RESOURCE_HOLY_POWER && result > 0 && talents.of_dusk_and_dawn->ok() &&
+       resources.current[ RESOURCE_HOLY_POWER ] == talents.of_dusk_and_dawn->effectN( 2 ).base_value() )
```

Comment:
this seems wrong - did you mean to delete the legendary check above, or to duplicate the whole block?

Reply:
The idea was to duplicate the whole block in order to keep it working during prepatch, could've done an OR check but decided against it in favour of being able to delete it easier once the legendary gets disabled.

Reply:
Right now it doesn't duplicate the whole block, just the condition, which is equivalent to `&&`-ing these conditions - you'll need to duplicate the `trigger` line as well

Reply:
Ah I didnt catch that, pushed a fix for that now :)